src/rpc/blockchain.cpp

```diff
@@ -1520,15 +1595,42 @@ static RPCHelpMan getmempoolinfo()
                         {RPCResult::Type::NUM, "maxmempool", "Maximum memory usage for the mempool"},
                         {RPCResult::Type::STR_AMOUNT, "mempoolminfee", "Minimum fee rate in " + CURRENCY_UNIT + "/kB for tx to be accepted. Is the maximum of minrelaytxfee and minimum mempool fee"},
                         {RPCResult::Type::STR_AMOUNT, "minrelaytxfee", "Current minimum relay fee for transactions"},
-                        {RPCResult::Type::NUM, "unbroadcastcount", "Current number of transactions that haven't passed initial broadcast yet"}
+                        {RPCResult::Type::NUM, "unbroadcastcount", "Current number of transactions that haven't passed initial broadcast yet"},
+                        {RPCResult::Type::OBJ, "fee_histogram", "",
+                            {
+                                {RPCResult::Type::OBJ_DYN, "ranges", "Feerate groups",
+                                {
+                                    {RPCResult::Type::OBJ, "<feerate-group>", "Object per feerate group",
+                                    {
+                                        {RPCResult::Type::NUM, "size", "Cumulated size of all transactions in feerate group"},
+                                        {RPCResult::Type::NUM, "count", "Amount of transactions in feerate group"},
+                                        {RPCResult::Type::NUM, "fees", "Cumulated fees of all transactions in feerate group (in satoshis)"},
+                                        {RPCResult::Type::NUM, "from_feerate", "Group contains transactions with feerates equal or greater than this value (in satoshis)"},
+                                        {RPCResult::Type::NUM, "to_feerate", "Group contains transactions with feerates less than this value (in satoshis)"},
+                                    }}}},
+                                {RPCResult::Type::NUM, "total_fees", "Total available fees in mempool (in satoshis)"},
```

Comment:
This field seems to be the same as the `total_fee` field above in the same output (except that it is now in sat/vB instead of BTC/kvB)?

Reply:
No. In short, it holds that `fee_histogram.total_fees <= total_fee`. More precisely, the `total_fees` (i.e. the histogram JSON property) here is a sum of `fees` JSON properties from fee rate groups (i.e. `ranges`). For example, `./bitcoin-cli getmempoolinfo "[0]"` should show the difference but not on testnet because people mostly do not pay any fees there (🙄).

However, I'm somewhat hesitant whether `total_fees` is useful or not. One can certainly compute it as it is a simple sum of already provided data. So it feels like I should remove it so that Bitcoin Core provides raw data and let customer applications do complex things based on the raw data. Of course, this is very subjective. I'm not sure. I'm neutral on this.